Quickstart tutorial (#802)
digital-asset/docs.daml.com: docs/2.10.0/docs/single-domain/extend-overview.rst

.. Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
.. SPDX-License-Identifier: Apache-2.0

Extend the Application
======================

> Alice can accept a transfer offer from the provider. However, it's possible that Alice, or any other user, may want to reject a transfer offer. For example, the provider may offer an incorrect amount, or the user may decide to reject the transfer for another reason. The remainder of this guide demonstrates how to extend the single-domain Daml application using an example scenario.

Reviewer: ```suggestion
Alice can accept a transfer offer from the provider. However, Alice, or any other user, may want to reject a transfer offer. For example, the provider may offer an incorrect amount, or the user may reject the transfer for another reason. The remainder of this guide demonstrates how to extend the single-domain Daml application to reject a transfer offer and record the reason for the rejection.
```
